content/guides/azure-todo-static-web-app.md

```diff
@@ -0,0 +1,747 @@
+---
+title: Building Azure Static Web Apps with Neon
+subtitle: A step-by-step guide to creating and deploying static sites using Azure and Neon Postgres
+author: dhanush-reddy
+enableTableOfContents: true
+createdAt: '2024-12-14T00:00:00.000Z'
+updatedOn: '2024-12-14T00:00:00.000Z'
+---
+
+If you've been looking for a modern approach to deploying web applications without managing traditional server infrastructure, [Azure Static Web Apps](https://azure.microsoft.com/en-us/products/app-service/static) might be exactly what you need. At its core, it's a service optimized for hosting static assets with global distribution, but its true power lies in the seamless integration with [Azure Functions](https://azure.microsoft.com/en-us/products/functions) for backend operations.
+
+What makes Azure Static Web Apps particularly compelling for developers is its built-in CI/CD pipeline powered by [Github Actions](https://github.com/features/actions). When you connect your repository, Azure automatically configures the necessary Github workflows – push your code and watch as Github Actions builds, optimizes, and deploys your entire application across a global network
+
+In this guide, we'll show you how to build a simple todo application using Azure Static Web Apps. You'll learn the basics of getting your website online and creating your first API endpoint with Azure Functions. By the end of this tutorial, you'll understand how to combine static web content with dynamic features to create a fully functional web application.
+
+## Prerequisites
+
+Before we begin, make sure you have:
+
+- [Node.js](https://nodejs.org/) 18.x or later installed
+- [Visual Studio Code](https://code.visualstudio.com/) with the [Azure Static Web Apps](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) and [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) extensions installed
```

Comment:
these are mentioned in prerequisties yet being installed in the guide.

Reply:
I considered omitting this from the prerequisites, but opted to include it for consistency like the guide here: https://neon.tech/guides/azure-functions-referral-system

Reply:
have removed from prerequisites